### PR TITLE
Env Var capture updates 

### LINKED ISSUE #3038: Add detail to env var capture page
We received some feedback about the env var page, and some minor updates including a new image are required.

https://elastic.slack.com/archives/CEV9CFY8H/p1678107298257599 for reference

### PR DESCRIPTION
**Pull request by benironside**
Fixes #3038 by adding information to the environment variable capture page that highlights how to view the captured data and provides an example image

Preview: [Environment variable capture ](https://security-docs_3063.docs-preview.app.elstc.co/guide/en/security/master/environment-variable-capture.html)

### CODE REVIEW

File: docs/cloud-native-security/environment-variable-capture.asciidoc [hunk 1]
```diff
@@ -20,12 +20,24 @@ You can configure an {agent} policy to capture up to five environment variables
...
 . Click *Show advanced settings*.
 . Scroll down or search for `linux.advanced.capture_env_vars`.
-. Enter the names of env vars you want to capture, separated by commas. For example: `PATH,LD_PRELOAD,USER`
+. Enter the names of env vars you want to capture, separated by commas. For example: `PATH,USER`
 . Click *Save*.
 
 [role="screenshot"]
 image::images/env-var-capture.png[The "linux.advanced.capture_env_vars" advanced agent policy setting]
+
+[discrete]
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/environment-variable-capture.asciidoc:33]:
```suggestion
[[find-cap-env-vars]]
[discrete]
```

File: docs/cloud-native-security/environment-variable-capture.asciidoc [hunk 2]
```diff
@@ -20,12 +20,24 @@ You can configure an {agent} policy to capture up to five environment variables
...
+== Find captured environment variables
+Captured environment variables are associated with process events, and appear in each event's `process.env_vars` field.
+
+This means that if you want to view environment variables in, for example, the *Events* table on the *Hosts* page, you can:
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/environment-variable-capture.asciidoc:37]:
```suggestion
To view environment variables in the *Events* table:
```

File: docs/cloud-native-security/environment-variable-capture.asciidoc [hunk 3]
```diff
@@ -20,12 +20,24 @@ You can configure an {agent} policy to capture up to five environment variables
...
+
+. Navigate to the table and click *Fields*.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/environment-variable-capture.asciidoc:39]:
```suggestion
. Click the *Event* tab on the *Hosts*, *Network*, or *User* pages, then click *Fields* in the Events table.
```

File: docs/cloud-native-security/environment-variable-capture.asciidoc [hunk 4]
```diff
@@ -20,12 +20,24 @@ You can configure an {agent} policy to capture up to five environment variables
...
+. Search for `process.env_vars`, select it, and click *Close*.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/environment-variable-capture.asciidoc:40]:
```suggestion
. Search for the `process.env_vars` field, select it, and click *Close*.
```

**Review by nick-alayil — APPROVED** [review]
LGTM. Do we want to explicitly mention that Hosts, Network or User pages is under Explore page(Security->Explore->Hosts/Network/User) similar to the way Security->Manage->Policies is mentioned for editing defend config?

File: docs/cloud-native-security/environment-variable-capture.asciidoc [hunk 5]
```diff
@@ -20,12 +20,25 @@ You can configure an {agent} policy to capture up to five environment variables
...
 image::images/env-var-capture.png[The "linux.advanced.capture_env_vars" advanced agent policy setting]
+
+[[find-cap-env-vars]]
+[discrete]
+== Find captured environment variables
+Captured environment variables are associated with process events, and appear in each event's `process.env_vars` field.
+
+To view environment variables in the *Events* table:
+
+. Click the *Event* tab on the *Hosts*, *Network*, or *User* pages, then click *Fields* in the Events table.
```
Comment (benironside) [inline docs/cloud-native-security/environment-variable-capture.asciidoc:40]:
```suggestion
. Click the *Event* tab on the *Hosts*, *Network*, or *Users* pages (*Security -> Explore*), then click *Fields* in the Events table.
```

File: docs/cloud-native-security/environment-variable-capture.asciidoc [hunk 6]
```diff
@@ -20,12 +20,25 @@ You can configure an {agent} policy to capture up to five environment variables
...
 image::images/env-var-capture.png[The "linux.advanced.capture_env_vars" advanced agent policy setting]
+
+[[find-cap-env-vars]]
+[discrete]
+== Find captured environment variables
+Captured environment variables are associated with process events, and appear in each event's `process.env_vars` field.
+
+To view environment variables in the *Events* table:
+
+. Click the *Event* tab on the *Hosts*, *Network*, or *Users* pages (*Security -> Explore*), then click *Fields* in the Events table.
```
Comment (benironside) [inline docs/cloud-native-security/environment-variable-capture.asciidoc:40]:
```suggestion
. Click the *Events* tab on the *Hosts*, *Network*, or *Users* pages (*Security -> Explore*), then click *Fields* in the Events table.
```

**Pull request merged**